base/src/main/java/io/spine/code/java/NestedClassName.java

```diff
@@ -43,4 +51,25 @@ static NestedClassName create(ClassName className) {
         String dotted = ClassName.toDotted(nameWithOuter);
         return new NestedClassName(dotted);
     }
+
+    /**
+     * Obtains this class name as a list of simple class names sorted in the nesting order.
+     *
+     * <p>For example, if this class name is {@code "Container.Job.Builder"}, then the resulting
+     * list would be {@code ["Container", "Job", "Builder"]}.
+     *
+     * <p>If this class name is just a {@link SimpleClassName}, then the only entry of the resulting
+     * list is that simple name.
+     *
+     * @return this name split into simple class names
+     */
+    public ImmutableList<SimpleClassName> asPath() {
```

Comment:
To me this is more like `split`. And `toPath()`should return the `Path` to declaring file.